docs/src/main/sphinx/connector/mysql.rst

```diff
@@ -71,11 +71,25 @@ Finally, you can access the ``clicks`` table in the ``web`` database::
 If you used a different name for your catalog properties file, use
 that catalog name instead of ``mysql`` in the above examples.
 
+
+Performance
```

Comment:
Fair enough .. This is in anticipation but I agree. Lets add when we have more. Ideally I would still like to have the Performance header just so that it is consistent across all connectors but we can do that in a follow up